src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,14 +1,37 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { actions as filterActions } from '../../features/filter';
+import { useAppSelector } from '../../app/hooks';
+import { Filter } from '../../types/Filter';
 
 export const TodoFilter: React.FC = () => {
+  const filters: Filter = useAppSelector(state => state.filter);
+
+  const dispatch = useDispatch();
+
+  const onSetStatus = (newStatus: string) => (
+    dispatch(filterActions.setStatus(newStatus))
+  );
+
+  const onSetQwery = (newQwery: string) => (
+    dispatch(filterActions.setQwery(newQwery))
+  );
```

Comment:
it's better to not store in redux variables which you use only in one component, you can store them in state